packages/astro/package.json

```diff
@@ -69,8 +69,7 @@
     "esbuild-node-externals": "^1.13.1",
     "execa": "^9.2.0",
     "typescript": "^5.4.5",
-    "vite-plugin-inspect": "0.8.4",
```

Comment:
This plugin has been pretty handy when debugging issues with virtual modules, it's probably fine if we keep it around?

Reply:
I would rather install it only when needed but sure I can revert this if we want to keep it around.

Reply:
Yep, the reason why I wanted to remove is this that you cannot use locally linked packages while running dev mode on TutorialKit monorepo. Linked projects will run into errors like `Cannot find package 'vite-plugin-inspect' imported from /x/tutorial-vite-plugin/node_modules/.pnpm/file+..+tutorialkit+packages+astro_@types+react-dom@18.3.0_astro@4.15.6_postcss@8.4.47_vite@5.4.5/node_modules/@tutorialkit/astro/dist/index.js`. But the solution for that is to not use dev/watch mode, and run manually `pnpm build` after making changes.

Reply:
1. Setup another Tutorial project with https://github.com/stackblitz/tutorialkit/blob/main/CONTRIBUTING.md#testing-tutorialkit-against-external-packages

2.1. Build linked monorepo with `pnpm dev`
2.2. Build or start dev mode of reference project
2.3. -> Error, no `vite-plugin-inspect`

3.1. Build linked monorepo with `pnpm build`
3.2. Build or start dev mode of reference project
3.3. -> Works fine

Reply:
Ah I see! Yeah that's annoying. Have you try doing:

```bash
rm -rf packages/template/src/content
ln -s ../path/to/other/tutorial/src/content ./packages/template/src/content
pnpm dev
```

If that works, maybe we could rename the `template/src/content` to something else and generate a symlink at build time (we could use `mklink` on Windows).

Alternatively, we could have this plugin behind a different environment variable that is not set by default (but could be set with a `.env` or equivalent).

Reply:
Linking just the `content` directories makes Astro throw `UnknownContentCollectionError` there.

Also with projects like Vite Plugin Tutorial it's not just as simple as linking the `content`. We would need templates, `astro.config.ts`, `theme.css`, override components, etc - pretty much whole project.

Reply:
Ok, let's put this import behind a different environment variable that has to be set manually then :+1:

Reply:
Added `VITE_INSPECT` for this.